cluster-up/hack/config.sh

```diff
@@ -2,7 +2,7 @@ unset docker_prefix master_ip network_provider kubeconfig manifest_docker_prefix
 
 KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-${PROVIDER}}
 
-source ${KUBEVIRTCI_PATH}hack/config-default.sh
+source ${KUBEVIRTCI_PATH}/hack/config-default.sh
```

Comment:
Why is this change required? FYI: The content of the folder `cluster-up` is taken from `kubevirt/kubevirtci`. Any errors should be fixed there.

Reply:
oh ok!